manifests/repository.pp

```diff
@@ -27,7 +27,7 @@
       apt::source { 'puppetlabs':
         location   => 'http://apt.puppetlabs.com',
         repos      => 'main',
-        key        => '4BD6EC30',
+        key        => '80F70E11F0F0D5F10CB20E62F5DA5F09C3173AA6',
```

Comment:
@tombuildsstuff This is the wrong fingerprint.
```47B320EB4C7C375AA9DA E1A01054B7A24BD6EC30``` is the longer fingerprint for the existing key, but this expires in March 2017
```6F6B 1550 9CF8 E59E 6E46  9F32 7F43 8280 EF8D 349F``` is the correct one, going by a Xenial deb package and https://docs.puppet.com/puppet/latest/puppet_collections.html#verify-the-fingerprint